src/components/Tabs/Tabs.tsx

```diff
@@ -12,15 +13,68 @@ interface ITabsProps {
 }
 
 export const Tabs = ({ children }: ITabsProps) => {
+  const [showScrollButton, setShowScrollButton] = useState(false);
+  const [disableScrollLeft, setDisableScrollLeft] = useState(true);
+  const [disableScrollRight, setDisableScrollRight] = useState(false);
```

Comment:
We agreed to use slightly different naming conventions for cases like this, `isScrollRightDisabled`, etc.

Also we use types `useState<boolean>`.

At same point we can revalidate our decision again, but for now we should be consistent.